tests/e2e/framework/kubernetes.go

```diff
@@ -487,7 +492,16 @@ func (k *KubeInfo) Teardown() error {
 		// clean up using helm
 		err := util.HelmDelete(istioHelmChartName)
 		if err != nil {
-			return nil
+			// If fail don't return so other cleanup activities can complete
+			log.Errorf("Helm delete of chart %s failed", istioHelmChartName)
+		}
+
+		if *useCNI {
+			err := util.HelmDelete("istio-cni")
+			if err != nil {
+				// If fail don't return so other cleanup activities can complete
```

Comment:
Don't we still need to fail at some point though? maybe use the multi errors?

Reply:
Added multierror then return